bin/add_uuid.rb

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+# Appends a UUID to each incoming holdings line
+
+require "securerandom"
+
+ARGF.each do |line|
+  if /^OCN\tBIB/.match?(line)
+    puts [line.strip, "UUID"].join("\t")
```

Comment:
The `line.strip` concerned me for a moment. I have confirmed holdings file lines will not end in an empty field because that is uniformly the GOV_DOC field which gets set one way or the other.

Reply:
Yes, it should be `chomp`